spec/acceptance/bundle_with_custom_path_spec.rb

```diff
@@ -57,8 +29,8 @@
       run 'bundle exec appraisal install'
 
       installed_gem = Dir.glob("tmp/stage/#{path}/ruby/*/gems/*").
-                      map    { |path| path.split('/').last }.
-                      select { |gem| gem.include?(gem_name) }
+        map { |path| path.split('/').last }.
+        select { |gem| gem.include?(gem_name) }
```

Comment:
We don't have Hound/automatic style linting on here, but I'd prefer the indentation stayed the same here.